src/Compilers/VisualBasic/Portable/Binding/MethodBodySemanticModel.vb

```diff
@@ -16,7 +16,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' <summary>
         ''' Creates an MethodBodySemanticModel that allows asking semantic questions about an attribute node.
         ''' </summary>
-        Friend Shared Function Create(binder As MethodBodyBinder, Optional ignoreAccessibility As Boolean = False) As MethodBodySemanticModel
+        Friend Shared Function Create(binder As SubOrFunctionBodyBinder, Optional ignoreAccessibility As Boolean = False) As MethodBodySemanticModel
```

Comment:
Simplifying the compiler by eliminating an unnecessary type. This changed line reflects the use of the base type instead of the type that has been eliminated.